shapely/geometry/linestring.py

```diff
@@ -39,11 +41,37 @@ def __init__(self, coordinates=None):
           >>> a.length
           2.0
         """
-        BaseGeometry.__init__(self)
-        if coordinates is not None:
-            ret = geos_linestring_from_py(coordinates)
-            if ret is not None:
-                self._geom, self._ndim = ret
+        if coordinates is None:
+            # empty geometry
+            # TODO better constructor
+            return pygeos.from_wkt("LINESTRING EMPTY")
+        elif isinstance(coordinates, LineString):
+            if type(coordinates) == LineString:
+                # TODO should we clone it?
```

Comment:
Just checked, eg copying a tuple (an immutable object) also returns the same object. So that seems like a common pattern. Will remove the comment then, and simply return the original object.